src/sm/resourcemanager/log.hpp

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LOG_HPP_
+#define LOG_HPP_
+
+#include "aos/common/tools/log.hpp"
+
+#define LOG_DBG() LOG_MODULE_DBG(LogModuleEnum::eSMResourceManager)
```

Comment:
That's should be clarified whether we need a separate eSMResourceManager in logs or a single LogModuleEnum::eSM per sm component would be enough.

Reply:
aos core zephyr has a separate enum value for the resource manager
https://github.com/aoscloud/aos_core_zephyr/blob/main/src/resourcemanager/log.hpp#L13
`Logger::Module::eResourceMgr`

That's why I've introduced a new one in the core lib cpp.

@al1img , can you please suggest on the approach? Should  I keep using the new enum values, or re-use LogModuleEnum::eSM?